hw12-webServer/src/main/java/ru/otus/kasymbekovPN/HW12/db/api/sessionManager/SessionManager.java

```diff
@@ -0,0 +1,35 @@
+package ru.otus.kasymbekovPN.HW12.db.api.sessionManager;
+
+import ru.otus.kasymbekovPN.HW12.db.hibernate.sessionManager.DataBaseSessionHibernate;
+
+/**
+ * Интерфейс для реализации менеджера сессий
+ */
+public interface SessionManager extends AutoCloseable{
+
+    /**
+     * Начать сессию
+     */
+    void beginSession();
+
+    /**
+     * Фиксировать изменения
+     */
+    void commitSession();
+
+    /**
+     * Откатить изменения
+     */
+    void rollbackSession();
+
+    /**
+     * Закрыить сессию
+     */
+    void close();
+
+    /**
+     * Геттер текущий сессии
+     * @return Текущая сессия
+     */
+    DataBaseSessionHibernate getCurrentSession();
```

Comment:
Почему не интерфейс?

```suggestion
    DataBaseSession getCurrentSession();
```